tests/integration/devfile/cmd_devfile_push_test.go

```diff
@@ -32,254 +32,199 @@ var _ = Describe("odo devfile push command tests", func() {
 		helper.CommonAfterEach(commonVar)
 	})
 
-	Context("verify devfile push works", func() {
-
-		It("checks that odo push works with a devfile", func() {
+	When("creating a nodejs compoenet", func() {
+		output := ""
+		BeforeEach(func() {
 			helper.Cmd("odo", "create", "nodejs", "--project", commonVar.Project, cmpName).ShouldPass()
-
 			helper.CopyExample(filepath.Join("source", "devfiles", "nodejs", "project"), commonVar.Context)
-			helper.CopyExampleDevFile(filepath.Join("source", "devfiles", "nodejs", "devfile.yaml"), filepath.Join(commonVar.Context, "devfile.yaml"))
+		})
+		When("setting git conifg", func() {
+			remoteURL := ""
```

Comment:
As its value does not change, you can affect the value directly here
```suggestion
			remoteURL := "https://github.com/odo-devfiles/nodejs-ex"